Npm install on windows fails

As this package is not currently available in the npm registry you have to do a direct install from github.

`npm install https://github.com/Polymer/polymer-decorators`

However when npm runs the **clean** script it fails on windows as the **rm** command is not available.

> @polymer/decorators@0.1.1 prepack C:\Users\user\AppData\Roaming\npm-cache\_cacache\tmp\git-clone-1748cdb6
> npm run build


> @polymer/decorators@0.1.1 build C:\Users\user\AppData\Roaming\npm-cache\_cacache\tmp\git-clone-1748cdb6
> npm run clean && tsc && rollup -c && node scripts/gen-global-typings.js


> @polymer/decorators@0.1.1 clean C:\Users\user\AppData\Roaming\npm-cache\_cacache\tmp\git-clone-1748cdb6
> rm -rf lib global.{js,d.ts} test/integration/{node_modules,bower_components,bower_cache}

> 'rm' is not recognized as an internal or external command,
operable program or batch file.
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! @polymer/decorators@0.1.1 clean: `rm -rf lib global.{js,d.ts} test/integration/{node_modules,bower_components,bower_cache}`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the @polymer/decorators@0.1.1 clean script.

Similar on linux, but it fails because the project doesn't has a `package.json`:

```
npm install https://github.com/Polymer/polymer-decorators                                                                                                                                                                       

> @polymer/decorators@0.1.1 prepack /home/adaliszk/.npm/_cacache/tmp/git-clone-c6e7b960
> npm run build

> @polymer/decorators@0.1.1 build /home/adaliszk/.npm/_cacache/tmp/git-clone-c6e7b960
> npm run clean && tsc && rollup -c && node scripts/gen-global-typings.js

> @polymer/decorators@0.1.1 clean /home/adaliszk/.npm/_cacache/tmp/git-clone-c6e7b960
> rm -rf lib global.{js,d.ts} test/integration/{node_modules,bower_components,bower_cache}

src/decorators.ts(12,23): error TS2688: Cannot find type definition file for 'reflect-metadata'.
src/decorators.ts(64,44): error TS2339: Property 'hasMetadata' does not exist on type 'typeof Reflect'.
src/decorators.ts(64,67): error TS2339: Property 'getMetadata' does not exist on type 'typeof Reflect'.
src/decorators.ts(65,17): error TS2339: Property 'hasMetadata' does not exist on type 'typeof Reflect'.
src/decorators.ts(66,22): error TS2339: Property 'getMetadata' does not exist on type 'typeof Reflect'.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @polymer/decorators@0.1.1 build: `npm run clean && tsc && rollup -c && node scripts/gen-global-typings.js`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the @polymer/decorators@0.1.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/adaliszk/.npm/_logs/2017-11-29T09_06_27_569Z-debug.log
npm ERR! code ENOPACKAGEJSON
npm ERR! package.json Non-registry package missing package.json: git+https://github.com/Polymer/polymer-decorators.git.
npm ERR! package.json npm can't find a package.json file in your current directory.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/adaliszk/.npm/_logs/2017-11-29T09_06_27_590Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
   "main": "lib/decorators.js",
   "types": "lib/decorators.d.ts",
   "scripts": {
-    "clean": "rm -rf lib polymer-decorators.{js,d.ts} mixins/*.d.ts test/integration/{node_modules,bower_components,bower_cache}",
+    "clean": "rimraf lib/ polymer-decorators.{js,d.ts} mixins/*.d.ts test/integration/{node_modules,bower_components,bower_cache}",
     "format": "find src test \\( -name node_modules -o -name bower_components -o -name bower_cache \\) -prune -o -name '*.ts' -print | xargs clang-format --style=file -i",
     "build": "npm run clean && tsc && rollup -c && node scripts/gen-global-typings.js && gen-typescript-declarations --outDir=.",
     "prepack": "npm run build",
@@ -27,6 +27,7 @@
   "devDependencies": {
     "@polymer/gen-typescript-declarations": "^1.1.3",
     "clang-format": "^1.0.55",
+    "rimraf": "^2.6.2",
     "rollup": "^0.55.0",
     "typescript": "^2.5.2"
   }

